projects.go

```diff
@@ -561,7 +561,7 @@ type EditProjectOptions struct {
 	Description                               *string             `url:"description,omitempty" json:"description,omitempty"`
 	IssuesAccessLevel                         *AccessControlValue `url:"issues_access_level,omitempty" json:"issues_access_level,omitempty"`
 	RepositoryAccessLevel                     *AccessControlValue `url:"repository_access_level,omitempty" json:"repository_access_level,omitempty"`
-	MergeRequestAccessLevel                   *AccessControlValue `url:"merge_request_access_level,omitempty" json:"merge_request_access_level,omitempty"`
+	MergeRequestAccessLevel                   *AccessControlValue `url:"merge_requests_access_level,omitempty" json:"merge_requests_access_level,omitempty"`
```

Comment:
I think we should change the field name as well right?
```suggestion
	MergeRequestsAccessLevel                  *AccessControlValue `url:"merge_requests_access_level,omitempty" json:"merge_requests_access_level,omitempty"`
```